api/http/handler/endpointproxy/proxy_docker.go

```diff
@@ -2,14 +2,12 @@ package endpointproxy
 
 import (
 	"errors"
-	"strconv"
-	"strings"
-	"time"
-
 	httperror "github.com/portainer/libhttp/error"
 	"github.com/portainer/libhttp/request"
 	"github.com/portainer/portainer/api"
 	bolterrors "github.com/portainer/portainer/api/bolt/errors"
+	"strconv"
+	"strings"
```

Comment:
this should be at the top - by convention all standard libs are the first imports. Something the IDE is doing?

Reply:
This is what my IDE does. I do not get why and how my IDE does the order.